modules/renter/downloadchunk.go

```diff
@@ -166,6 +171,24 @@ func (udc *unfinishedDownloadChunk) returnMemory() {
 	}
 }
 
+// addChunkToCache adds the chunk to the cache if the download is a streaming
+// endpoint download.
+func (udc *unfinishedDownloadChunk) addChunkToCache(data []byte) {
```

Comment:
This should probably live in cache.go - we aren't sure how long term the cache is going to be, we should make it easy to track it down and remove it, or upgrade it.